kclvm/tools/src/LSP/src/completion.rs

```diff
@@ -37,8 +36,34 @@ pub(crate) fn completion(
     if let Some('.') = trigger_character {
         completion_dot(program, pos, prog_scope)
     } else {
-        // todo: Complete identifiers such as attr, variables, types, etc.
-        None
+        let mut completions: IndexSet<String> = IndexSet::new();
+
+        // complete all usable scope obj in inner_most_scope
+        if let Some(inner_most_scope) = prog_scope.inner_most_scope(pos) {
```

Comment:
It also work in schema stmt, if stmt and lambda rule, I will add some ut about it. 
It need to distinguishes whether in schema expr because need to find schema attr name to complete, which not contained in current scope. In other cases, we can find complete items in scope.